data-studio/README.md

```diff
@@ -0,0 +1,115 @@
+## Create reports and dashboards with Data Studio
+
+![](images/example_dashboard.png)
+
+This is a short tutorial on how to set up a dashboard (or report as stated by
+data studio) in Google Data Studio (this can easily be done in other tools such
+as [Tableu](https://www.tableau.com/)).
+
+The aim is to show the potential of what you can do with the data from our
+Wittra sensor network.
+
+### Prerequisites
+
+In order to produce the dashboard it assumes you have already set up a
+database, to store the streamed data from the network. In this example we use
+[BigQuery](https://cloud.google.com/bigquery/what-is-bigquery), an enerprise
+data warehouse by Google. For inspiration on how to setup a database from
+streamed data please see [this example](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#running-the-pubsub-topic-to-bigquery-template)
+
+A Google account, to log into Data Studio. Note that Data Studio is free, but
+the queries to the data base may cost money depending on the database and the
+amount of data.
+
+### 1. Create a new dashboard
+
+Start by logging into
+[Data Studio](https://datastudio.google.com/navigation/reporting) and click the
+Blank Report button.
+
+![](images/datastudio_start.png)
+
+### 2. Add a data source
+
+To populate the dashboard you will need to add a *connector* (in this case your
+BigQuery). When creating a new report the `add data view` will be opened
+automatically, if not click `add data` to the right in the top panel.
+
+![](images/datastudio_add_data.png)
+
+There are several different type of connectors already existing in Data Studio,
+some are directly connected to the Google infrastructure and some are external
+connectors such as *MySQL*, *PostgreSQL* and *File Upload*. You
+can also [add your own connectors](https://developers.google.com/datastudio/connector/build)
+to interface with Data Studio.
+In this example we will create a dashboard for a specific tag showing
```

Comment:
Either break the line once more to create a new paragraph, or concatenate with previous line to keep it in one. :slightly_smiling_face: